tests/foreman/cli/test_activationkey.py

```diff
@@ -1575,3 +1575,106 @@ def test_positive_copy_subscription(self):
             subscription_result.stdout[0]['name'],  # subscription name
             result.stdout[3]  # subscription list
         )
+
+    def test_positive_update_auto_attach(self):
+        """@Test: Update Activation key with inverse auto-attach value
+
+        @Feature: Activation key update / info
+
+        @Steps:
+        1. Get the key's current auto attach value.
+        2. Update the key with the value's inverse.
+        3. Verify key was updated.
+
+        @Assert: Activation key is sucessfully copied
+
+        """
+        org_id = make_org(cached=True)['id']
+        key_id = make_activation_key(
+            {u'organization-id': org_id}, cached=True)['id']
+        try:
+            attach_value = ActivationKey.info({
+                u'id': key_id,
+                u'organization-id': org_id
+            }).stdout['auto-attach']
+        except CLIFactoryError as err:
+            self.fail(err)
+        # invert value
+        new_value = u'false' if attach_value == u'true' else u'true'
+        try:
+            result = ActivationKey.update({
+                u'id': key_id,
+                u'organization-id': org_id,
+                u'auto-attach': new_value
+            })
+            self.assertEqual(result.return_code, 0)
+            attach_value = ActivationKey.info({
+                u'id': key_id,
+                u'organization-id': org_id
+            }).stdout['auto-attach']
+        except CLIFactoryError as err:
+            self.fail(err)
+
+        self.assertEqual(attach_value, new_value)
+
+    @data(
+        u'1',
+        u'0',
+        u'true',
+        u'false',
+        u'yes',
+        u'no',
+    )
+    def test_positive_update_auto_attach_2(self, new_value):
+        """@Test: Update Activation key with valid auto-attach values
+
+        @Feature: Activation key update / info
+
+        @Steps:
```

Comment:
Add a blank line between `@steps` and the list. Without that line, this is considered one big paragraph:

    @Steps: 1. Update the key with a valid value 2. Verify key was updated